core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -325,9 +325,11 @@ class RequestChannel(val numProcessors: Int, val queueSize: Int) extends KafkaMe
   }
 
   def sendShutdownRequest(): Unit = requestQueue.put(ShutdownRequest)
+
 }
 
 object RequestMetrics {
+  val metricsMap = new scala.collection.mutable.HashMap[String, RequestMetrics]
```

Comment:
Is this intentional?

Reply:
It's not. Probably a result of a wrong conflict resolve.